modules/core/src/main/scala/com/azavea/stac4s/extensions/eo/package.scala

```diff
@@ -0,0 +1,13 @@
+package com.azavea.stac4s.extensions
+
+import eu.timepit.refined._
+import eu.timepit.refined.api._
+import eu.timepit.refined.numeric._
+
+package object eo {
+  type Percentage = Int Refined Interval.Closed[W.`0`.T, W.`100`.T]
+  object Percentage extends RefinedTypeOps[Percentage, Int]
+
+  type BandRange = Int Refined Interval.Closed[W.`0`.T, W.`100`.T]
```

Comment:
Is there a reason to limit this to 100? I think maybe this should be non-negative - some [hyperspectral](https://gisgeography.com/multispectral-vs-hyperspectral-imagery-explained/) imagery can have 100s of bands.

Reply:
I would say that it should be any positive number; who knows what standards may appear...

Reply:
It was a "big number." Since geotiff-has-band-x is really hard for us to verify (and a tif at a given location can change contents), I capped it at "so many bands." I'll uncap it though.

Reply:
0f0427f